docs/HelpWantedProcess.md

```diff
@@ -0,0 +1,13 @@
+This document describes the process that the team uses for identifying candidate issues for community members to tackle.
+
+## Process
+
+This process (we'll refer to it as `help-wanted` process going forward) triggers from the moment, when the `help-wanted` label is assigned to an issue.
+At that time, the team who is triaging the issue, also assigns an engineer to the issue.
+The task for the assigned engineer will be to understand the ask based on all the comments in the issue and summarize the problem in a comment using the [Help Wanted Template](/docs/HelpWantedIssueSummaryCommentTemplate.md).
+As part of filling out the template, the engineer will also need to describe a high-level design for how to approach the problem and how it can be solved.
+When all the sections of the template are filled in, the user will then need to apply one of the `Complexity: ` labels, to indicate how easy / hard will it be to tackle the issue.
+This will help community members to find the right type of issue to contribute to.
+After posting this comment, the assigned engineer can unassign themself from the issue.
+
+This will conclude the process.
```

Comment:
Is there some expectation that they will own reviewing any PRs that may result?

Reply:
I don't want to introduce constraints here. Part of the reason is that the current process for handling community PRs works relatively good today. The second part is that we don't know when someone from the community will decide to pick up the help-wanted issue. And by the time they do, the original person who dropped the comment may not even be on the team.

Reply:
Perhaps we could say something like "The engineer who filled out the template is not committed to reviewing any resulting PRs"?

Reply:
Or simply not mention it at all. I really don't want to tie these two processes together. That said, I'll monitor this comment to see if others feel strong about this. And if more people think it's important to make this explicit, then I'll mention it.